Orange/widgets/visualize/owscatterplot.py

```diff
@@ -179,22 +182,36 @@ def __init__(self):
         self.sampling.setVisible(False)
 
         box = gui.vBox(self.controlArea, "Points")
+        color_model = DomainModel(
+            order=("(Same color)", dmod.Separator) + dmod.SEPARATED,
+            valid_types=dmod.PRIMITIVE)
         self.cb_attr_color = gui.comboBox(
             box, self, "graph.attr_color", label="Color:",
             emptyString="(Same color)", callback=self.update_colors,
```

Comment:
I hated this repetition, too.

In my first implementation, the default was the first string value in `order`. It had a similar effect as what you propose, but does not introduce yet another argument, and it also works if you want to put the default at the end. But I didn't do it since perhaps sometimes there is no default, and the first value is just a string.

I don't like your solution any better because it automagically adds something to `order`. Also, I have no way to decide whether I'd like to have it separated or not.

I currently see no better solution than `emptyString`, so for now I'm keeping what we had before.